stylesheet.css

```diff
@@ -0,0 +1,45 @@
+#main {
+  background-color: rgb(238, 238, 238);
+  border-radius: 2px;
+}
+
+body {
+  text-align: center;
+  font-size: 100%;
+  font-family: apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+}
+
+ul {
+  max-width: 550px;
+  margin: auto;
+  text-align: left;
+}
+
+#image {
+  margin: auto;
+  font-size: 100%;
+  max-width: 97%;
+  display: block;
+  padding: 7px 17px;
+}
+
+#img-caption {
+  background-color: white;
+  text-align: center;
+  margin: 1px 7px;
+}
+
+#img-div {
+  background-color: white;
+}
+
+blockquote {
+  font-size: 16px;
+  text-align: center;
+  margin-left: 20%;
```

Comment:
We don't need a `margin-left` there, this element should align with the rest of the list item. See screenshot below:

![image](https://user-images.githubusercontent.com/36057474/147683884-9dc6b4aa-6303-41fd-918c-d2c3b7684d40.png)